cves/CVE-2009-3548.yml

```diff
@@ -43,29 +43,36 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  The Windows installer for Apache Tomcat uses a blank default 
+  password for the administrative user, which allows remote attackers to gain privileges easily.
+  This is obviously aggergious because it gives anyone admin access without having to
+  provide any credentials.
 
 bounty_instructions: |
   If you came across any indications that a bounty was paid out for this
   vulnerability, fill it out here. Or correct it if the information already here
   was wrong. Otherwise, leave it blank.
 bounty:
   amt:
-  announced: 2009-11-12
+  announced:
   url:
-bugs: []
+bugs: [36954]
 fixes_vcc_instructions: |
   Please put the Git commit SHA in "commit" below, and any notes about how this
   was discovered in the "note" field.
   Refer to our instructions on how to find a Git SHA from an SVN revision.
 fixes:
    - commit: d48de20a81d5796db9677b54ff971ad9757fc158
      note: SVN rev 919006, from the Tomcat website.
-   - commit:
-     note:
+   - commit: cb8855a65c580e332681f61a7d6edfa509588bf0
+     note: Github revision, found through tomcat by following date of fix
 vccs:
-  - commit:
-    note:
+  - commit: 7f6d0018bbb36b63f4651e453be61ab298993297
+    note: |
+      rmaucher added entire file (res/tomcat.nsi) during commit with message 
+      "More release scripts work" on July 12 2006. So it appears this vulnerability was 
+      introduced upon file inception.
```

Comment:
Could use some context on who rmaucher is as this person is not previously mentioned

Reply:
I will update with some background info on who this person is within the organization.